doajtest/unit/test_journalcsv.py

```
@@ -266,5 +254,148 @@ def test_04_background_task(self):
             assert csv_rows[1] != csv_rows[2]             # ID was changed, rows should differ
 
             # check that the difference between rows is the ISSN
+            assert diff(csv_rows[2], csv_rows[1]) == ["8765-4321"]
 
-            assert diff(csv_rows[2], csv_rows[1]) == ["8765-4321"]
\ No newline at end of file
+    def test_05_mixed_unicode(self):
```

I might be mistaken, but I'm not seeking any actual non-ascii data despite seeing some unicode strings. Those may silently convert (successfully) to normal strings without errors where non-ascii strings will not. Try setting the journal's title to `u"Заглавие на журнала"` then running the CSV export and checking for it.